core/common/src/test/java/alluxio/AlluxioTestDirectory.java

```diff
@@ -90,12 +81,19 @@ private static void cleanUpOldFiles(File dir) {
     File[] files = dir.listFiles();
     for (File file : files) {
       if (!FileUtils.isFileNewer(file, cutoffTimestamp)) {
-        try {
-          alluxio.util.io.FileUtils.deletePathRecursively(file.getAbsolutePath());
-        } catch (Exception e) {
-          LOG.warn("Failed to delete {} : {}", file.getAbsolutePath(), e.getMessage());
-        }
+        delete(file);
```

Comment:
When do we decide to use `stream`?

Reply:
my current rule is to use stream whenever it seems cleaner and performance isn't critical. Updated this.